### PR TITLE
Login Function

### PR DESCRIPTION
**Pull request by REFAEI91**

### CODE REVIEW

File: styles.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default createGlobalStyle`
   *::before,
   *::after {
     box-sizing: border-box;
+    /* outline: 1px solid black; */
```
Comment (MerleMak) [inline styles.js:14]: you can delete commented out code before committing :)

File: components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      </>
+    );
+  }
+  return (
+    <>
+      <Button onClick={() => signIn()}>
+        <Image src="/login.svg" width={32} height={32} alt="login" />
+        Sign in
+      </Button>
+    </>
```
Comment (MerleMak) [inline components/Login/Login.js]:
```suggestion
      <Button onClick={() => signIn()}>
        <Image src="/login.svg" width={32} height={32} alt="login" />
        Sign in
      </Button>
```

File: components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+export default function Login() {
+  const { data: session } = useSession();
+  if (session) {
+    return (
+      <>
+        <Button onClick={() => signOut()}>
+          <Image src="/logout.svg" width={32} height={32} alt="logout" />
+          Sign out
+        </Button>
+      </>
```
Comment (MerleMak) [inline components/Login/Login.js]:
```suggestion
        <Button onClick={() => signOut()}>
          <Image src="/logout.svg" width={32} height={32} alt="logout" />
          Sign out
        </Button>
```

File: components/Login/Login.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    );
+  }
```
Comment (MerleMak) [inline components/Login/Login.js]: you could add an `else` here :)

**Review by MerleMak — COMMENTED** [review]
Looks really good! I left some comment for you, that you might want to consider. Really nice work! Keep it up 🚀

File: styles.js (hunk 1 above)
Comment (REFAEI91) [reply styles.js:14]: Thank you for your feedback i will delete it now .

File: components/Login/Login.js (hunk 4 above)
Comment (REFAEI91) [reply components/Login/Login.js]: Thank you for your feedback i will add it now

**Comment by REFAEI91** [comment]
Thank you for your feedback! I appreciate your positive comments and I'm glad you think the work looks good. I have already made all the suggested changes you mentioned and incorporated them into the final version. Thank you once again for your support and encouragement! @MerleMak

**Pull request merged**